src/main/java/caculator/enums/Operator.java

```diff
@@ -0,0 +1,52 @@
+package caculator.enums;
+
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", 1, (firstOperand, secondOperand) -> firstOperand + secondOperand ),
+    MINUS("-", 1, (firstOperand, secondOperand) -> firstOperand - secondOperand ),
+    MULTIPLY("*", 2, (firstOperand, secondOperand) -> firstOperand * secondOperand ),
+    DIVIDE("/", 2, (firstOperand, secondOperand) -> {
+        if(secondOperand == 0)
+            throw new RuntimeException("0으로 나눌 수 없습니다.");
+        return firstOperand / secondOperand;
+    });
+
+    private final String operator;
+    private final Integer priority;
+    private final BiFunction<Double, Double, Double> expression;
+
+    Operator(String operator, Integer priority, BiFunction<Double, Double, Double> expression) {
+        this.operator = operator;
+        this.priority = priority;
+        this.expression = expression;
+    }
+
+    public static Operator findOperator(String value) {
+        if(value.equals(PLUS.operator))
+            return PLUS;
+        if(value.equals(MINUS.operator))
+            return MINUS;
+        if(value.equals(MULTIPLY.operator))
+            return MULTIPLY;
+        if(value.equals(DIVIDE.operator))
+            return DIVIDE;
+        throw new IllegalArgumentException();
+    }
+
+    public Integer getPriority() {
+        return priority;
+    }
+
+    public Double operate(Double firstOperand, Double secondOperand) {
+        return expression.apply(firstOperand, secondOperand);
+    }
+
+    public boolean isSameOperator(String operator) {
+        return this.operator.equals(operator);
+    }
```

Comment:
```suggestion
    public boolean isSameOperator(String operator) {
        return Arrays.stream(values())
	        .anyMatch(op -> op.operator.equals(operator));
    }
```

이 메서드를 Util에서 가져가서 로직을 만드는데, enum 내에서 구현해도 되지 않을까 싶습니다.
boolean 값을 반환하는 anyMatch를 사용하게 되면, 위에서 제안해 드렸던 연산자 수정 또는 추가 시
enum 내에서 연산자만 추가하면 그에 따른 로직이 알아서 처리될 듯합니다.

Reply:
넵 enum 내부에서 처리하는 편이 나을 것 같습니다! 수정하겠습니다!

Reply:
https://github.com/programmers-lecture/java-calculator/pull/6/commits/a7baed38dac5a2f61e8af302b905ad6bd03c40b9
stream 사용하여 수정하면서 enum 내부에서 처리하도록 했습니다.